File: Tests/CK.StObj.TypeScript.Tests/CrisLike/Model/ICommandAuthCritical.cs
using CK.Core;

#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved

namespace CK.StObj.TypeScript.Tests.CrisLike
{
    /// <summary>
    /// Extends <see cref="ICommandAuthNormal"/> to ensure that the authentication level is <see cref="AuthLevel.Critical"/>.
    /// </summary>
    [CKTypeDefiner]
    public interface ICommandAuthCritical : ICommandAuthNormal
    {
    }
}

File: Tests/CK.TypeScript.Tests/TSInlineTests/CrisLike_commands_and_results/ck-gen/CK/CrisLike/ICommandAuthCritical.ts
import { ICommandAuthNormal } from './ICommandAuthNormal';

/**
 * Extends ICommandAuthNormal to ensure that the authentication level is 
 * ~~!:AuthLevel.Critical~~.
 **/
export interface ICommandAuthCritical extends ICommandAuthNormal {
readonly _brand: ICommandAuthNormal["_brand"] & {"27":any};
}
